src/ParticipantClient.ts

```diff
@@ -26,14 +26,12 @@ export class ParticipantClient extends Client {
 
     public open(options: IParticipantOptions): Promise<this> {
         return super.open({
-            jwt: options.jwt,
             url: options.url,
-            queryParams: Object.assign(
-                {
-                    'x-protocol-version': '2.0',
-                },
-                options.extraParams,
-            ),
+            queryParams: {
+                'x-protocol-version': '2.0',
+                key: options.key,
+                ...options.extraParams,
+            },
```

Comment:
Opted to go this route instead of introducing a top-level property into the socket; I don't believe the socket should necessarily know how we want to authenticate.